modules/instrument_builder/jsx/react.questions.js

```diff
@@ -631,6 +631,15 @@ var AddElement = React.createClass({
         });
         hasError = true;
       }
+      if (minYear > 9999 || minYear < 1900 || maxYear > 9999 || maxYear < 1900) {
+        let temp = (this.state.error) ? this.state.error : {};
+
+        temp.dateOption = "The year must have exactly 4 digits. Please choose an integer number between 1900 and 9999.";
```

Comment:
could we add an 'e' after 'befor' on line 628? (https://github.com/kongtiaowang/Loris/blob/33ed933d2a665b4636c20242d033e410f5069ddd/modules/instrument_builder/jsx/react.questions.js#L628) i'm commenting that here because i can't comment on that line

Reply:
Excellent catch!